src/features.xml

```diff
@@ -100,6 +100,11 @@
         <description>Enables the dynamic profile generator for OpenSSH config files</description>
         <id>9031</id>
         <stage>AlwaysDisabled</stage>
+        <alwaysEnabledBrandingTokens>
+            <brandingToken>Dev</brandingToken>
+            <brandingToken>Canary</brandingToken>
+            <brandingToken>Preview</brandingToken>
+        </alwaysEnabledBrandingTokens>

```

Comment:
Seems like this should be `AlwaysEnabled` then?

Reply:
Nope! The SSH code is hidden behind a feature flag on `Stable` builds so I'm making sure we still don't expose it yet. I want to make sure the experience feels right with the Extensions page. Once the Extensions page goes to `Stable`, we can just remove the feature flag altogether.

`WindowsInbox` is also one that's omitted. Same deal there.